src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss

```diff
@@ -0,0 +1,14 @@
+.MyAccountPasswordForm {
+    &-Wrapper {
+        display: flex;
+        justify-content: center;
+        @include after-mobile {
+            padding-top: .5rem; // var(--popup-content-padding) - 1rem
```

Comment:
I am unsure on necessity of styles above. Please check how other popups are styled. I think there is some trick to it :thinking: